packages/react-components/react-migration-v0-v9/src/components/Attachment/AttachmentAction.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button, ButtonProps, mergeClasses } from '@fluentui/react-components';
+import * as React from 'react';
+import { useAttachmentActionStyles } from './AttachmentAction.styles';
+
+export type AttachmentActionProps = ButtonProps;
+
+export const attachmentActionClassName = 'fui-AttachmentAction';
+
+export const AttachmentAction = React.forwardRef<HTMLButtonElement, AttachmentActionProps>((props, ref) => {
+  const { className, disabled, disabledFocusable, children, onClick, onKeyUp, onKeyDown, ...rest } = props;
+  const classes = useAttachmentActionStyles();
+
+  const handleClick = React.useCallback(e => {
+    e.stopPropagation();
+    e.preventDefault();
+    onClick?.(e);
+  }, [onClick]);
+
+  const handleKeyUp = React.useCallback(e => {
+    if (e.key === 'Enter' || e.key === ' ') {
+      e.stopPropagation();
+    }
+    onKeyUp?.(e);
+  }, [onKeyUp]);
+
+  const handleKeyDown = React.useCallback(e => {
+    if (e.key === 'Enter' || e.key === ' ') {
+      e.stopPropagation();
+    }
+    onKeyDown?.(e);
+  }, [onKeyDown]);

```

Comment:
```suggestion
```

From what I see the v0 implementation does not stop keyboard events bubbling to the parent attachment from the action

Reply:
v0 implementation does not use this keydown/keyup combination but useARIAButton does - without it pressing Enter was causing attachment onClick to be called - so I stopped it here and now it works.